Model/src/BoardPackage/PropertySquare.java

```diff
@@ -1,7 +1,287 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
-public class PropertySquare {
+import GamePackage.Player;
+
+/********************************************************************
+ * The PropertySquare class is a BoardSquare class that defines the
+ * BoardSquares that are properties. It tracks variables related to
+ * properties such as the PRICE of purchase and the rent cost of
+ * landing on this BoardSquare.
+ *
+ * @author David Baas
+ * @version 2/12/2018
+ ********************************************************************/
+public class PropertySquare extends BoardSquare{
+
+    /**An int to hold the PRICE of the property*/
+    private final int PRICE;
+
+    /**An int to hold the property's mortgage value*/
+    private final int MORTGAGE_VAL;
+
+    /**An int to hold the cost of rent w/ no houses.*/
```

Comment:
These cannot be finals because we said the deck class will instantiate all their values from a text file or something.

Reply:
Even when creating a property from a file using a different class (such as the Deck Class), the constants can still be there. Making these values constant means that once the instance of a property is created and the values are set, they can't be changed again. Nor do they need to be changed again. The only values that have to change are the ones that regard the owner of the property, weather it's mortgaged or not, and how many houses are on the property.